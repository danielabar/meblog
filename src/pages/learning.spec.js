import React from "react"
import { render, screen } from "@testing-library/react"
import "@testing-library/jest-dom"

import Learning from "./learning"

jest.mock("../components/SEO", () => {
  return {
    __esModule: true,
    default: () => {
      return <div></div>
    },
  }
})

const courseData = {
  allMarkdownRemark: {
    totalCount: 1,
    edges: [
      {
        node: {
          id: "aaa111",
          frontmatter: {
            category: "Linux",
            completed_date: "2023-03-31",
            description: "Course description blah blah",
            featuredImage: {
              childImageSharp: {
                gatsbyImageData: {
                  layout: "constrained",
                  backgroundColor: "#282838",
                  images: {
                    fallback: {
                      src: "/static/abc123/5a159/bash.jpg",
                      srcSet:
                        "/static/abc123/ba115/bash.jpg 63w,\n/static/abc123/1a003/bash.jpg 125w,\n/static/abc123/5a159/bash.jpg 250w,\n/static/abc123/d3119/bash.jpg 500w",
                      sizes: "(min-width: 250px) 250px, 100vw",
                    },
                    sources: [
                      {
                        srcSet:
                          "/static/abc123/a6e40/bash.webp 63w,\n/static/abc123/46142/bash.webp 125w,\n/static/abc123/2cd09/bash.webp 250w,\n/static/abc123/cd07d/bash.webp 500w",
                        type: "image/webp",
                        sizes: "(min-width: 250px) 250px, 100vw",
                      },
                    ],
                  },
                  width: 250,
                  height: 140,
                },
              },
            },
            instructor: "John Doe",
            notes: "https://github.com/danielabar/bash-zsh-syntax-pluralsight",
            platform: "Pluralsight",
            title: "Mastering Bash and Z Shell Scripting Syntax",
          },
        },
      },
    ],
  },
}

describe("Learning Page", () => {
  it("renders", () => {
    const container = render(<Learning data={courseData} />)
    expect(container).toMatchSnapshot()
    expect(screen.getByTestId("learning")).toBeInTheDocument()
  })
})
