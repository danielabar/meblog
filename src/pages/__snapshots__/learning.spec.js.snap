// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Learning Page renders 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <header
          data-testid="header"
        >
          <div>
            <a
              href="/"
            >
              <div>
                <div>
                  <img
                    alt="Profile"
                    src="/images/dbaron_profile.png"
                  />
                </div>
                <div
                  class="undefined undefined"
                >
                  Daniela Baron
                </div>
              </div>
            </a>
            <nav
              data-testid="nav-menu"
            >
              <ul>
                <li
                  class="undefined undefined"
                >
                  <a
                    href="/blog"
                  >
                    Blog
                  </a>
                </li>
                <li
                  class="undefined undefined"
                >
                  <a
                    href="/learning"
                  >
                    Learning
                  </a>
                </li>
                <li
                  class="undefined undefined"
                >
                  <a
                    href="/about"
                  >
                    About
                  </a>
                </li>
                <li
                  class="undefined undefined"
                >
                  <a
                    href="https://danielabaron.me/rss.xml"
                  >
                    RSS
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </header>
        <div>
          <div
            data-testid="learning"
          >
            <div />
            <section
              data-testid="learning-intro"
            >
              Welcome to the Learning section. In a world where change is the only constant, it's crucial to keep learning and expanding our knowledge. This is especially true in software development, where new languages, tools, libraries, and frameworks are always emerging. Here, I've gathered a collection of online courses that I've taken and compiled detailed notes on, so you can learn from my experiences and take your skills to the next level. Also check out my article on how to
               
              <a
                class="markdown-link"
                href="https://danielabaron.me/blog/learning-from-screencasts/"
              >
                get the most out of online courses
              </a>
              , and start your learning journey today!
            </section>
            <section
              data-testid="course-list"
            >
              <div>
                <div>
                  <div
                    class="gatsby-image-wrapper gatsby-image-wrapper-constrained"
                    data-gatsby-image-wrapper=""
                    style="position: relative; overflow: hidden; display: inline-block; vertical-align: top;"
                  >
                    <div
                      style="max-width: 250px; display: block;"
                    >
                      <img
                        alt=""
                        aria-hidden="true"
                        role="presentation"
                        src="data:image/svg+xml;charset=utf-8,%3Csvg%20height='140'%20width='250'%20xmlns='http://www.w3.org/2000/svg'%20version='1.1'%3E%3C/svg%3E"
                        style="max-width: 100%; display: block; position: static;"
                      />
                    </div>
                    <div
                      aria-hidden="true"
                      data-placeholder-image=""
                      style="height: 100%; left: 0px; position: absolute; top: 0px; width: 100%;"
                    />
                    <picture>
                      <source
                        data-srcset="/static/abc123/a6e40/bash.webp 63w,/static/abc123/46142/bash.webp 125w,/static/abc123/2cd09/bash.webp 250w,/static/abc123/cd07d/bash.webp 500w"
                        sizes="(min-width: 250px) 250px, 100vw"
                        type="image/webp"
                      />
                      <img
                        alt="Mastering Bash and Z Shell Scripting Syntax"
                        data-gatsby-image-ssr=""
                        data-main-image=""
                        data-src="/static/abc123/5a159/bash.jpg"
                        data-srcset="/static/abc123/ba115/bash.jpg 63w,/static/abc123/1a003/bash.jpg 125w,/static/abc123/5a159/bash.jpg 250w,/static/abc123/d3119/bash.jpg 500w"
                        decoding="async"
                        loading="lazy"
                        sizes="(min-width: 250px) 250px, 100vw"
                        style="height: 100%; left: 0px; position: absolute; top: 0px; transform: translateZ(0); transition: opacity 250ms linear; width: 100%; will-change: opacity; opacity: 0;"
                      />
                    </picture>
                    <noscript />
                    <script
                      type="module"
                    >
                      const t="undefined"!=typeof HTMLImageElement&&"loading"in HTMLImageElement.prototype;if(t){const t=document.querySelectorAll("img[data-main-image]");for(let e of t){e.dataset.src&&(e.setAttribute("src",e.dataset.src),e.removeAttribute("data-src")),e.dataset.srcset&&(e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset"));const t=e.parentNode.querySelectorAll("source[data-srcset]");for(let e of t)e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset");e.complete&&(e.style.opacity=1,e.parentNode.parentNode.querySelector("[data-placeholder-image]").style.opacity=0)}}
                    </script>
                  </div>
                </div>
                <div
                  class="undefined Linux"
                >
                  <div>
                    <span>
                      •
                    </span>
                     
                    Linux
                  </div>
                </div>
                <div>
                  Mastering Bash and Z Shell Scripting Syntax
                </div>
                <div>
                  <div>
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M400 64h-48V12c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v52H160V12c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v52H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V112c0-26.5-21.5-48-48-48zm-6 400H54c-3.3 0-6-2.7-6-6V160h352v298c0 3.3-2.7 6-6 6z"
                      />
                    </svg>
                    <span>
                      2023-03-31
                    </span>
                  </div>
                  <div>
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                      />
                    </svg>
                    <a
                      data-testid="course-notes-link"
                      href="https://github.com/danielabar/bash-zsh-syntax-pluralsight"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      View Notes
                    </a>
                  </div>
                </div>
              </div>
            </section>
            <div
              data-testid="all-wrapper"
              style="margin-top: 30px;"
            >
              <a
                href="/blog"
              >
                All Articles
              </a>
            </div>
          </div>
        </div>
        <footer
          data-testid="footer"
        >
          <hr />
          <p>
            All materials © Daniela Baron 2025
          </p>
          <div>
            <a
              href="https://twitter.com/DanielaMBaron"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                />
              </svg>
            </a>
            <a
              href="https://github.com/danielabar"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 496 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                />
              </svg>
            </a>
            <a
              href="https://linktr.ee/danielabaron"
            >
              <svg
                fill="currentColor"
                height="1em"
                role="img"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title />
                <path
                  d="m13.73635 5.85251 4.00467-4.11665 2.3248 2.3808-4.20064 4.00466h5.9085v3.30473h-5.9365l4.22865 4.10766-2.3248 2.3338L12.0005 12.099l-5.74052 5.76852-2.3248-2.3248 4.22864-4.10766h-5.9375V8.12132h5.9085L3.93417 4.11666l2.3248-2.3808 4.00468 4.11665V0h3.4727zm-3.4727 10.30614h3.4727V24h-3.4727z"
                />
              </svg>
            </a>
            <a
              href="https://www.linkedin.com/in/danielabaron/"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
                />
              </svg>
            </a>
          </div>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <header
        data-testid="header"
      >
        <div>
          <a
            href="/"
          >
            <div>
              <div>
                <img
                  alt="Profile"
                  src="/images/dbaron_profile.png"
                />
              </div>
              <div
                class="undefined undefined"
              >
                Daniela Baron
              </div>
            </div>
          </a>
          <nav
            data-testid="nav-menu"
          >
            <ul>
              <li
                class="undefined undefined"
              >
                <a
                  href="/blog"
                >
                  Blog
                </a>
              </li>
              <li
                class="undefined undefined"
              >
                <a
                  href="/learning"
                >
                  Learning
                </a>
              </li>
              <li
                class="undefined undefined"
              >
                <a
                  href="/about"
                >
                  About
                </a>
              </li>
              <li
                class="undefined undefined"
              >
                <a
                  href="https://danielabaron.me/rss.xml"
                >
                  RSS
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </header>
      <div>
        <div
          data-testid="learning"
        >
          <div />
          <section
            data-testid="learning-intro"
          >
            Welcome to the Learning section. In a world where change is the only constant, it's crucial to keep learning and expanding our knowledge. This is especially true in software development, where new languages, tools, libraries, and frameworks are always emerging. Here, I've gathered a collection of online courses that I've taken and compiled detailed notes on, so you can learn from my experiences and take your skills to the next level. Also check out my article on how to
             
            <a
              class="markdown-link"
              href="https://danielabaron.me/blog/learning-from-screencasts/"
            >
              get the most out of online courses
            </a>
            , and start your learning journey today!
          </section>
          <section
            data-testid="course-list"
          >
            <div>
              <div>
                <div
                  class="gatsby-image-wrapper gatsby-image-wrapper-constrained"
                  data-gatsby-image-wrapper=""
                  style="position: relative; overflow: hidden; display: inline-block; vertical-align: top;"
                >
                  <div
                    style="max-width: 250px; display: block;"
                  >
                    <img
                      alt=""
                      aria-hidden="true"
                      role="presentation"
                      src="data:image/svg+xml;charset=utf-8,%3Csvg%20height='140'%20width='250'%20xmlns='http://www.w3.org/2000/svg'%20version='1.1'%3E%3C/svg%3E"
                      style="max-width: 100%; display: block; position: static;"
                    />
                  </div>
                  <div
                    aria-hidden="true"
                    data-placeholder-image=""
                    style="height: 100%; left: 0px; position: absolute; top: 0px; width: 100%;"
                  />
                  <picture>
                    <source
                      data-srcset="/static/abc123/a6e40/bash.webp 63w,/static/abc123/46142/bash.webp 125w,/static/abc123/2cd09/bash.webp 250w,/static/abc123/cd07d/bash.webp 500w"
                      sizes="(min-width: 250px) 250px, 100vw"
                      type="image/webp"
                    />
                    <img
                      alt="Mastering Bash and Z Shell Scripting Syntax"
                      data-gatsby-image-ssr=""
                      data-main-image=""
                      data-src="/static/abc123/5a159/bash.jpg"
                      data-srcset="/static/abc123/ba115/bash.jpg 63w,/static/abc123/1a003/bash.jpg 125w,/static/abc123/5a159/bash.jpg 250w,/static/abc123/d3119/bash.jpg 500w"
                      decoding="async"
                      loading="lazy"
                      sizes="(min-width: 250px) 250px, 100vw"
                      style="height: 100%; left: 0px; position: absolute; top: 0px; transform: translateZ(0); transition: opacity 250ms linear; width: 100%; will-change: opacity; opacity: 0;"
                    />
                  </picture>
                  <noscript />
                  <script
                    type="module"
                  >
                    const t="undefined"!=typeof HTMLImageElement&&"loading"in HTMLImageElement.prototype;if(t){const t=document.querySelectorAll("img[data-main-image]");for(let e of t){e.dataset.src&&(e.setAttribute("src",e.dataset.src),e.removeAttribute("data-src")),e.dataset.srcset&&(e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset"));const t=e.parentNode.querySelectorAll("source[data-srcset]");for(let e of t)e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset");e.complete&&(e.style.opacity=1,e.parentNode.parentNode.querySelector("[data-placeholder-image]").style.opacity=0)}}
                  </script>
                </div>
              </div>
              <div
                class="undefined Linux"
              >
                <div>
                  <span>
                    •
                  </span>
                   
                  Linux
                </div>
              </div>
              <div>
                Mastering Bash and Z Shell Scripting Syntax
              </div>
              <div>
                <div>
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 448 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M400 64h-48V12c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v52H160V12c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v52H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V112c0-26.5-21.5-48-48-48zm-6 400H54c-3.3 0-6-2.7-6-6V160h352v298c0 3.3-2.7 6-6 6z"
                    />
                  </svg>
                  <span>
                    2023-03-31
                  </span>
                </div>
                <div>
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                    />
                  </svg>
                  <a
                    data-testid="course-notes-link"
                    href="https://github.com/danielabar/bash-zsh-syntax-pluralsight"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    View Notes
                  </a>
                </div>
              </div>
            </div>
          </section>
          <div
            data-testid="all-wrapper"
            style="margin-top: 30px;"
          >
            <a
              href="/blog"
            >
              All Articles
            </a>
          </div>
        </div>
      </div>
      <footer
        data-testid="footer"
      >
        <hr />
        <p>
          All materials © Daniela Baron 2025
        </p>
        <div>
          <a
            href="https://twitter.com/DanielaMBaron"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
          </a>
          <a
            href="https://github.com/danielabar"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 496 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
              />
            </svg>
          </a>
          <a
            href="https://linktr.ee/danielabaron"
          >
            <svg
              fill="currentColor"
              height="1em"
              role="img"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title />
              <path
                d="m13.73635 5.85251 4.00467-4.11665 2.3248 2.3808-4.20064 4.00466h5.9085v3.30473h-5.9365l4.22865 4.10766-2.3248 2.3338L12.0005 12.099l-5.74052 5.76852-2.3248-2.3248 4.22864-4.10766h-5.9375V8.12132h5.9085L3.93417 4.11666l2.3248-2.3808 4.00468 4.11665V0h3.4727zm-3.4727 10.30614h3.4727V24h-3.4727z"
              />
            </svg>
          </a>
          <a
            href="https://www.linkedin.com/in/danielabaron/"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
              />
            </svg>
          </a>
        </div>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
